.github/workflows/icon-generation.yaml

```diff
@@ -0,0 +1,67 @@
+name: Update Icons
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update-icons:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Use pnpm
+        uses: pnpm/action-setup@v3
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: "pnpm"
+
+      - name: Install dependencies
+        run: pnpm i --frozen-lockfile
+
+      - name: Build icons
+        run: pnpm build:icons
+        env:
+          FIGMA_ACCESS_TOKEN: ${{ secrets.FIGMA_ACCESS_TOKEN }}
+
+      - name: Generate icons
+        run: pnpm --stream --filter {packages/fondue} generate:icons
+
+      - name: Create Unique Changeset
```

Comment:
since this is created, won't it mean that `git-check.outputs.changes` is always true?

Reply:
Yup José is right, we need create the changeset file if « git status » returns you any changes (the build script will add/modify/delete files if there are changes)

Reply:
sounds good, will reorder